lib/pymedphys/docs/release-notes.md

```diff
@@ -46,7 +46,8 @@ This project adheres to
 ### New features and enhancements
 
 * Added CLI argument for setting the hostname on the DICOM listen server. For
-  example `pymedphys dicom listen --host 127.0.0.1`.
+  example `pymedphys dicom listen --host 127.0.0.1 7779`.
```

Comment:
Does the `7779` need to be proceeded with `--port`?

Reply:
No, since thats not an optional parameter.. The user must specify a port...

Reply:
Yup, fair enough :slightly_smiling_face: 

Could we change the text to the following:

```suggestion
  example `pymedphys dicom listen 7779 --host 127.0.0.1 `.
```

Optional parameters at the end tend to make more sense to me.

Reply:
Yep, that is nicer.